src/relayclient/ContractInteractor.ts

```diff
@@ -26,10 +28,12 @@ let IRelayHubContract: Contract<IRelayHubInstance>
 let IForwarderContract: Contract<ITrustedForwarderInstance>
 
 export default class ContractInteractor {
+  private readonly web3: Web3
   private readonly provider: provider
   private readonly config: ContractInteractorConfig
 
   constructor (provider: provider, config: ContractInteractorConfig) {
+    this.web3 = new Web3(provider)
```

Comment:
you actually took most references to web3 out. maybe it can be left confined to the interactor...

Reply:
I do intend to remove all references to web3 from anywhere but interactor, and rename it BlockchainInteractor.

Reply:
... and then maybe use "ethers", AFAIK its ligther than web3...